ArcadiaParties.CQRS/Queries/GetUserFromAssistantQuery.cs

```diff
@@ -0,0 +1,9 @@
+﻿using ArcadiaParties.Data.Abstractions.DTOs;
+using MediatR;
+
+namespace ArcadiaParties.CQRS.Queries
+{
+    public class GetUserFromAssistantQuery : IRequest<UserFromAssistantDTO>
```

Comment:
The same here. `GetAssistantUserQuery` and `AssistantUserDTO` sound better.

Reply:
Renamed